### PR TITLE
Create kingston.md

### PR DESCRIPTION
**Pull request by MattCookEmpire**

### CODE REVIEW

File: citmtc/kingston.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Kingston
+
+### Restaurants :pizza:
+- bubba's
+- Jack Astor's
+- Pilot House
```
Comment (margaretupshur) [inline citmtc/kingston.md:6]: What do they serve at Pilot House? @citmtc

File: citmtc/kingston.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Kingston
+
+### Restaurants :pizza:
+- bubba's
```
Comment (JNKRMN) [inline citmtc/kingston.md:4]: @citmtc Bubba gumps?

File: citmtc/kingston.md (hunk 1 above)
Comment (Nickismyname) [inline citmtc/kingston.md:6]: HI @citmtc, is this seafood?

**Comment by JNKRMN** [comment]
Looks good to me (LGTM) merge this ish!

File: citmtc/kingston.md (hunk 1 above)
Comment (MattCookEmpire) [inline citmtc/kingston.md:6]: @Nickismyname Not exactly seafood...but famous for fish & chips!  And beer.

Comment (MattCookEmpire) [inline citmtc/kingston.md:6]: @margaretupshur mostly fish and chips...and beer! :+1:

**Pull request merged**